### ISSUE TITLE
Missing parameter destroys following

### ISSUE THREAD

**Issue by tvld** [adnanh/webhook#232]
This does not work, $1 and $2 empty:
```
 "pass-arguments-to-command":
    [
      {
        "source": "payload",
        "name": "actor.username"
      },
      {
        "source": "string",
        "name": "/home/myself/website"
      }
    ]
```
This works, $1 has value, $2 empty:
```
"pass-arguments-to-command":
    [
      {
        "source": "string",
        "name": "/home/myself/website"
      },
     {
        "source": "payload",
        "name": "actor.username"
      },
    ]
```

**Comment by adnanh** [comment]
Which version of webhook are you using and what is the payload you are sending the hook. Sounds like an old bug we fixed where parameter extraction would stop after first fail (which explains your examples).

**Comment by tvld** [comment]
I am running the default webhook from the Ubuntu ppa, like a good citizen ;) 
How to determine? ``webhook -version`` does not show me anything, and ``webhook -help`` shows no version output?

**Comment by adnanh** [comment]
Yeah, then you're running a 2.5.x which was affected with the issue #106.

The issue has been fixed in 2.6.0, I highly recommend downloading the latest version from the [releases](https://github.com/adnanh/webhook/releases) page.

Prior to the 2.6.0 the only way to determine the version was to run the webhook with `-verbose` flag and to check stdout.

**Comment by tvld** [comment]
Thank you... wish the ppa was updated though... :
